app/helpers/notifications_helper.rb

```diff
@@ -78,6 +90,22 @@ def mark_read_selected_button
     function_button('Mark as read', 'eye', 'mark_read_selected', 'Mark items as read')
   end
 
+  def subscribe_repo_button
+      button_tag(type: 'button', id: "#{filters[:repo]}", class: "subscribe_repo btn btn-default", 'data-toggle': "tooltip", 'data-placement': "bottom", 'title': 'Subscribe to this repository' ) do
+      octicon('unmute', height: 16) + content_tag(:span, " Subscribe repository", class: 'hidden-xs')
+    end
+  end
+
+  def unsubscribe_repo_button
+  #  function_button('Unsubscribe repository', 'circle-slash', 'unsubscribe_repo', 'Unsubscribe this repository')
+    button_tag(type: 'button', id: "#{filters[:repo]}", class: "unsubscribe_repo btn btn-default", 'data-toggle': "tooltip", 'data-placement': "bottom", 'title': 'Unsubscribe this repository' ) do
+      octicon('mute', height: 16) + content_tag(:span, " Unsubscribe repository", class: 'hidden-xs')
+    end
+
+  end
+
+
+
```

Comment:
Can you remove two of these extra newlines?

Reply:
Done.